conda/plugins/types.py

```diff
@@ -13,14 +13,16 @@ class CondaSubcommand(NamedTuple):
     :param name: Subcommand name (e.g., ``conda my-subcommand-name``).
     :param summary: Subcommand summary, will be shown in ``conda --help``.
     :param action: Callable that will be run when the subcommand is invoked.
+    :param no_sys_argv: Do not pass sys.argv values.
     """
 
     name: str
     summary: str
     action: Callable[
-        [list[str]],  # arguments
+        ...,  # arguments
         int | None,  # return code
     ]
+    no_sys_argv: bool = False
```

Comment:
Double-negative which looks weird when writing it as `no_sys_argv=True`, should be:
```suggestion
    pass_sys_argv: bool = True
```

Reply:
Thanks for catching this. Will change 👍